variables.tf

```diff
@@ -222,12 +222,6 @@ variable "kms_key_deletion_window" {
 
 # Secrets Manager
 # ---------------
-variable "tfe_license_secret" {
-  type = object({
-    arn = string
-  })
-  description = "The Secrets Manager secret under which the Base64 encoded Terraform Enterprise license is stored."
-}
```

Comment:
This input is still necessary, though we should reconsider if it (and `ca_certificate_secret`) should be simple strings rather than objects.